horace_core/horace_function_utils/horace_function_call_method.m

```diff
@@ -93,19 +93,19 @@
 end
 
 if all(w.sqw_type)
-    dummy_obj=sqw_old;
+    dummy_obj=sqw;
 else
     ndims=w.ndims(1);
     if ndims==0
-        dummy_obj=d0d_old;
+        dummy_obj=d0d;
```

Comment:
do these work?

This function (horace_function_call_method) should be disappearing as part of the DnD migration

Reply:
I think this was just a tidy up on my part. I did not test functionality outside their use (if any) in the existing tests. As I made the change, it must have fixed one or more tests.